hack/sast.sh

```diff
@@ -10,13 +10,17 @@ root_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
 
 gosec_report="false"
 gosec_report_parse_flags=""
+exclude_dirs="hack"
```

Comment:
In https://github.com/gardener/gardener-extension-registry-cache/pull/272#discussion_r1824125810 I suggested to make the output file of the report configurable. WDYT?